.travis.yml

```diff
@@ -3,16 +3,16 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "6"
+  - "10"
 
-sudo: false
 dist: trusty
 
 addons:
   chrome: stable
 
 cache:
-  yarn: true
```

Comment:
Seems the `yarn` based setup was lost here somehow. But I wouldn't care about that, as I would like to modernize more of the addon's setup in the near future (e.g. migrate to Github actions)

Reply:
oh if you want yarn that should be easy enough to add back. I used ember-cli-update to upgrade the testing infrastructure and it usually it notices that you're using yarn and does the right thing.

It looks like you don't have a yarn.lock file committed, if we do that I think ember-cli-update will start doing the right thing again 👍